src/http.rs

```diff
@@ -375,7 +401,12 @@ where
 
 /// Register a WebSockets path with the HTTP server. Your app must do this
 /// in order to receive incoming WebSocket connections.
-pub fn bind_ws_path<T>(path: T, authenticated: bool, encrypted: bool) -> anyhow::Result<()>
+pub fn bind_ws_path<T>(
+    path: T,
+    authenticated: bool,
+    encrypted: bool,
+    extension: bool,
+) -> anyhow::Result<()>
```

Comment:
In this case I'd prefer to keep the main bind function with the old signature, and make a new function called `bind_extension_path` or something like that.